README.md

```diff
@@ -93,6 +100,10 @@ Nginx Configuration:
 * `NGINX_MAX_CLIENT_BODY_SIZE`: Configures the max client request body size of nginx. _(Default: `0`, Disables checking of client request body size.)_
 * `NGINX_STATUS_PATH`: Change the nginx status path that returns 200 on the default server. (Default: `/dispatcher/status`)
 * `PORT`: This is the port that nginx will listen on _(Default: `80`)_
+* `SSL_PORT`: This is the SSL port nginx will listen on for ssl enabled hosts. _(Default: `443`)_
+* `SSL_CERT`: Path to a ssl certificate used for the default ssl nginx server. Enables SSL support.
+* `SSL_KEY`: Path to a ssl certificate key used for the default ssl nginx server.
+* `SSL_CERT_DIR`: Path to write namespace host's certificates to. _(Default: `/etc/nginx/ssl`)_
```

Comment:
Are you referring to `SSL_CERT_DIR` specifically or all SSL_* config options. (GH makes it hard to tell sometimes.)

In our deployment we own the Docker container but other people using it may have different needs i guess. I added it initially so i could test easily without running dispatcher as root and having access to `/etc/nginx`. I don't feel to opinionated on the matter though so i could remove it.